src/cast_core/test/CrudActorYamlTests.yml

```diff
@@ -434,7 +434,16 @@ Tests:
       - Filter: {a: 1}
         Count: 0
 
-  - Description: simple findOneAndUpdate
+  - Description: simple findOne
+    Operations:
+      - OperationName: insertOne
+        OperationCommand:
+          Document: {a: 100}
+      - OperationName: findOne
+        OperationCommand:
+          Filter: {a: 100}
+
+- Description: simple findOneAndUpdate
```

Comment:
ugh lololol yaml - the tests are failing because this line (446) needs to be indented a couple spaces

Reply:
Thanks. I pushed a fix. My yamllint is all very unhappy with that file anyways, so I didn't see the warning on it. Have you considered a yaml lint task?

Reply:
Hm, still failing. I tried a couple of things also. Could you take a look again @rtimmons?